crates/biome_cli/tests/snapshots/main_cases_protected_files/not_process_file_from_cli_verbose.snap

```diff
@@ -21,6 +21,10 @@ internalError/io ━━━━━━━━━━━━━━━━━━━━━
 
 # Emitted Messages
 
+```block
+Scanned project folder in <TIME>.
+```
```

Comment:
Scanning was enabled for formatting too in this PR. It's in anticipation for monorepo support, since scanning for nested `biome.json` files would rely on the same scanning mechanism.